run-tests-with-build-optawebs-parent/optawebs-project-sources-test/pom.xml

```diff
@@ -12,6 +12,14 @@
 
     <artifactId>optawebs-project-sources-test</artifactId>
 
+    <properties>
+        <sources.directory>${sources.directory.module.project.sources.dependency.get}</sources.directory>
+        <openshift.api-url>https://api.production.rhba.openshift-psi.rhocf-dev.net:6443</openshift.api-url>
```

Comment:
should this URL be published?

Reply:
It could not, but I leave it here just for simplification since this address is not going to change frequently

Reply:
Fixed